python/python/glide/async_commands/utils/utils.py

```diff
@@ -0,0 +1,58 @@
+from typing import Any, Dict, Mapping, Optional, Union
+
+from glide.constants import TClusterDecodedResponse, TClusterResponse
+
+
+def convert_bytes_to_string_dict(
+    byte_string_dict: Optional[Union[Mapping[bytes, Any], Dict[bytes, Any]]]
+) -> Optional[Dict[str, Any]]:
+    """
+    Recursively convert the keys and values of a dictionary from byte strings to regular strings,
+    handling nested dictionaries of any depth.
+
+    Args:
+        byte_string_dict (Optional[Union[Mapping[bytes, Any], Dict[bytes, Any]]]):
+        A dictionary where keys and values can be byte strings or nested dictionaries.
```

Comment:
Please check and align doc style thoughout
```suggestion
            A dictionary where keys and values can be byte strings or nested dictionaries.
```